doc/Doxyfile

```diff
@@ -780,9 +790,7 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = ./
-INPUT                 += doc/main.md
-USE_MDFILE_AS_MAINPAGE = doc/main.md
```

Comment:
doesn't this remove the main.md from the doxygen?

Reply:
Yeah but this page is not used by Sphinx-exhale, it uses `index.rst`.